qa/tasks/cephfs/test_volume_client.py

```diff
@@ -19,7 +19,8 @@ class TestVolumeClient(CephFSTestCase):
     def setUp(self):
         CephFSTestCase.setUp(self)
         self.py_version = self.ctx.config.get('overrides', {}).get('python', 'python')
-        log.info("using python version: %s".format(self.py_version))
+        log.info("using python version: {python_version}".format(
```

Comment:
```suggestion
        log.info("using python version: %r", self.py_version)
```

Reply:
just curious: is the suggested only for having narrower lines/conciseness or are there more advantages to it?